mk/submodule.mk

```diff
@@ -26,3 +26,19 @@ $(LIBTUV_PATH)/include:
 
 $(LIBTUV_LIBRARY):
 	$(MAKE) -C $(LIBTUV_PATH) TUV_BUILD_TYPE=release TUV_CREATE_PIC_LIB=yes TUV_PLATFORM=$(LIBTUV_PLATFORM) TUV_BOARD=$(LIBTUV_BOARD)
+
+# librabbitmq related variables
+LIBRABBITMQ_PATH = deps/rabbitmq-c
+LIBRABBITMQ_INCLUDE := -I $(LIBRABBITMQ_PATH)/build/include
+LIBRABBITMQ_LIBRARY := $(LIBRABBITMQ_PATH)/build/librabbitmq/librabbitmq.a
+
+$(LIBRABBITMQ_PATH)/build/include:
+	git submodule update --init $(LIBRABBITMQ_PATH)
+	mkdir $(LIBRABBITMQ_PATH)/build
+	cd $(LIBRABBITMQ_PATH)/build && \
+         cmake -DCMAKE_INSTALL_PREFIX=. .. && \
```

Comment:
@marktwtn, `cmake` is indeed one of the build dependencies. We shall mention it in documentation. Can you send pull request for it?

Reply:
Sure~

Reply:
I will send a pull request after this one is merged.

Reply:
I'll start to prepare the pull request since `libtuv` requires `cmake`, too.

Reply:
Done in the commit 07fa452.